jboss-cert-ldap-login-module/pom.xml

```diff
@@ -0,0 +1,90 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.esbtools.auth</groupId>
+    <artifactId>cert-ldap-login-module</artifactId>
+    <version>1.3.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>jboss-cert-ldap-login-module</artifactId>
+
+  <repositories>
+    <repository>
+      <id>jboss</id>
+      <url>https://repository.jboss.org/nexus/content/repositories/releases/</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.esbtools.auth</groupId>
+      <artifactId>org.esbtools.auth.common</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.spec</groupId>
+      <artifactId>jboss-javaee-6.0</artifactId>
+      <version>3.0.2.Final</version>
+      <type>pom</type>
+      <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <artifactId>xalan</artifactId>
+          <groupId>xalan</groupId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.picketbox</groupId>
+      <artifactId>picketbox</artifactId>
+      <version>4.9.6.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    
+    <!-- EAP versions -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.7</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>2.6</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Even if we did an import, it would lock us to a specific version of EAP dependencies, which I am not a fan of. I would like to just delete this block and force consuming applications to deal with specifics if needed.

Reply:
No idea if there is a pom to import these versions from somewhere (probably not).  

Not sure if I understand how "delete this block and force consuming applications to deal with specifics if needed" would actually work since the project would no longer compile and/or generate runtime errors if you did that.  

Perhaps I am not fully understanding what you're suggesting

Reply:
I got a little carried away. Looking again, it is only slf4j that could be removed as it is defined in the parent pom at version 1.7.25. The other dependencies would need to remain for exactly the reason that you stated. Sorry.

Reply:
As for dealing with specifics, I guess my question is what version of EAP is this targeting? Different versions of EAP are going to provide different versions of these dependencies, which has the potential to cause runtime errors. I was suggesting we remove the provided scope, use the latest and greatest versions if we choose, and let consumers dependencyManage if needed, rather than trying to guess what they might need.

Reply:
Okay, you bring up a good point about the versions though...I agree the with your suggestion that removing provided and using latest and greatest is probably best bet here.  I can't remember why these got marked as provided in the first place, let's make sure to test this in JBoss when it is finished to make sure removing provided doesn't cause runtime issues

Reply:
Ideally this should work in either version of EAP (which I think it does currently)

Reply:
> I got a little carried away.

Or did I?

commons-lang and guava are defined in the common library at versions 2.6 and 19.0 respectively. So the redefinition in jboss-cert-ldap-login-module is only to enforce a version and provided scope. Which seems ugly to me.

Reply:
If we're rolling with non-provided versions, then they can/should only exist in common I would think